### ISSUE TITLE
Additional CSS in Site Editor encodes characters to HTML

### ISSUE THREAD

**Issue by metabreakr** [Automattic/wp-calypso#74906]
### Quick summary

When adding a chevron `>` in Additional CSS in the Site Editor, it is encoded as `&gt;` after saving, which breaks the CSS code. For example:

```
.wp-site-blocks > * + * {
	margin-block-start: 0;
}
```

...becomes:

```
.wp-site-blocks &gt; * + * {
	margin-block-start: 0;
}
```

### Steps to reproduce

1. Go to _Appearance → Editor → Styles_
2. Select "Additional CSS"
3. Paste code (such as the above) and save
4. Revisit the site editor, and observe that the code has been altered

### What you expected to happen

I expected `>` to remain a chevron character.

### What actually happened

The `>` chevron became `&gt;` instead.

### Impact

All

### Available workarounds?

Yes, easy to implement

### Platform (Simple and/or Atomic)

Simple

### Logs or notes

This is reproducible on simple, not sure about Atomic, and wasn't an issue with previous Gutenberg. The workaround is to use the Customizer to add CSS in the meantime.

**Comment by cuemarie** [comment]
📌 SCRUBBING : RESULT - *Replicated / Could Not Replicate / Uncertain*
- Tested on Simple: **Replicated**
- Tested on AT: Could not replicate 
- Tested on Self-hosted: Could not replicate 
- Replicable on Core? No

📌 FINDINGS/SCREENSHOTS/VIDEO
- Replicated on Simple site only, on themes Zoologist & Twenty Twenty-Three
     - the change was visible on screen after saving - no need to refresh page.
- On AT and Self-Hosted, the `>` chevron remained as expected.

**Simple Example**


https://user-images.githubusercontent.com/27249804/228676282-aa909065-3ac0-4b87-b14c-9324ff36afc5.mov

📌 ACTIONS
- Assigned to T-Rex

**Comment by mmtr** [comment]
Fixed in D109069-code
